Dockerfile.jenkins

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:8-jre-alpine
+WORKDIR /root
+COPY app.jar .
```

Comment:
Due to this line in particular, I renamed this to be Docker.jenkins since it has a tight dependency on the actual logic of the pipeline script.